ios/Podfile

```diff
@@ -14,6 +14,8 @@ target 'growupmobile' do
 
   pod 'react-native-image-picker', :path => '../node_modules/react-native-image-picker'
 
+  pod 'react-native-simple-toast', :path => '../node_modules/react-native-simple-toast'
```

Comment:
It look like this package does not allow agile customisation. Let's change this to react-native-toast-message instead.